nilearn/decoding/decoder.py

```diff
@@ -1264,6 +1290,9 @@ def __sklearn_tags__(self):
         tags.estimator_type = "classifier"
         tags.classifier_tags = ClassifierTags()
 
+        # TODO
+        # check sklearn ClassifierMixin to adapt this
```

Comment:
Not sure I understand this

Reply:
Will open a separate issue to discuss and track and will suggest a PR to show what could be done.

Reply:
See #5583